modular_citadel/code/game/objects/ids.dm

```diff
@@ -43,4 +43,32 @@
 
 /obj/item/card/id/knight/examine(mob/user)
 	..()
-	to_chat(user, "<span class='notice'>Alt-click to recolor it.</span>")
\ No newline at end of file
+	to_chat(user, "<span class='notice'>Alt-click to recolor it.</span>")
+
+//=================================================
+
+/obj/item/card/emag_broken
+	desc = "It's a card with a melted magnetic strip, useless!"
+	name = "broken cryptographic sequencer"
+	icon_state = "emag"
+	item_state = "card-id"
+	lefthand_file = 'icons/mob/inhands/equipment/idcards_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/idcards_righthand.dmi'
+	flags_1 = NOBLUDGEON_1
+	flags_2 = NO_MAT_REDEMPTION_2
+	color = rgb(35, 20, 11)
+
+/obj/item/card/emag
+	var/uses = 10
+
+/obj/item/card/emag/afterattack(atom/target, mob/user, proximity)
+	. = ..()
+	uses--
+
+	if(uses<1)
```

Comment:
I know this was already merged, but uh, I'm going to nitpick anyway:

```
uses = max(0, uses - 1)

if(!uses)
```

would have been better logic for this.